clients/webclient/src/components/Users/CustomerEdit.vue

```diff
@@ -0,0 +1,139 @@
+<template>
+    <v-form>
+        <v-container>
+            <v-row>
+                <v-col>
+                    <h1>Endre profil</h1>
+                </v-col>
+            </v-row>
+            <v-row class="text-center">
+                <v-col>
+                    <v-text-field
+                        label="Fullt navn"
+                        v-model="fullname"
+                    ></v-text-field>
+                </v-col>
+            </v-row>
+            <v-row class="text-center">
+                <v-col>
+                    <v-text-field
+                        label="Adresse"
+                        v-model="address"
+                    ></v-text-field>
+                </v-col>
+            </v-row>
+            <v-row class="text-center">
+                <v-col cols="4">
+                    <v-text-field
+                        :rules="[numbers, phoneNoLength]"
+                        label="Telefonnummer"
+                        v-model="phone"
+                        required
+                    ></v-text-field>
+                </v-col>
+                <v-col>
+                    <v-text-field
+                        label="E-post"
+                        v-model="email"
+                        required
+                    ></v-text-field>
+                </v-col>
+            </v-row>
+            <v-row>
+                <v-col :cols="2">
+                    <p class="font-weight-black">
+                        <label for="allergylist">Mine allergier</label>
+                    </p>
+                </v-col>
+                <v-col>
+                    <p
+                        class="font-weight-light"
+                        v-for="allergy in allergies"
+                        :key="allergy"
+                        id="allergylist"
+                    >
+                        {{ allergy }}
+                    </p>
+                </v-col>
+            </v-row>
+            <v-row>
+                <v-col>
+                    <v-btn @click="sendToDb" color="primary" class="ma-1">
+                        Send inn
+                    </v-btn>
+                    <v-btn color="secondary" @click="cancel" class="ma-1">
+                        Avbryt
+                    </v-btn>
+                </v-col>
+            </v-row>
+        </v-container>
+    </v-form>
+</template>
+
+<script lang="ts">
+import Vue from "vue";
+import Component from "vue-class-component";
+import { Prop } from "vue-property-decorator";
+import api from "../../api/api";
+import { Userprofile } from "../../../../../server/src/interfaces";
+import CustomerProfile from "../Users/CustomerProfile.vue";
+
+@Component({
+    components: {
+        CustomerProfile,
+    },
+})
+export default class CustomerEdit extends Vue {
+    @Prop() loggedInUser!: string;
+    @Prop() userprofile!: Userprofile;
+    private fullname = "";
```

Comment:
I stedet for å kopiere ut disse properties som separate variabler er det bedre å refere direkte til userprofile i template: userprofile.fullname, userprofile.phone etc. Da verdiene automatisk oppdateres andre steder i appen.